docs/source/external/matrix.rst

```diff
@@ -0,0 +1,37 @@
+MATRIX
+======
+
+
+Post-processing grids
+---------------------
+
+We need to post-process the raw grids outputted by MATRIX -
+use one of the available interfaces (including the CLI) to do so.
+
+
+- **Optimize and scale:** apply the built-in optimization and rescaled by a factor if needed to go e.g. from pb to fb.
+
+- **Delete keys:** delete `y_label_unit`, `xlabel` and `x_label-tex`
+
+- **Add keys:**
+    .. list-table:: Additional keys
+        :widths: 25 25 50
+        :header-rows: 1
+
+        * - key
+          - value
+          - comment
+        * - y-unit
+          - pb/GeV
+          -
+        * - y-label
+          - differential x-sec
+          -
+        * - lumi_id_types
+          - pdg_mc_ids
```

Comment:
again you want to write "since MATRIX is not doing this by default, remember to add it" and then complain with MATRIX